source/includes/kubernetes/_k8_storageclasses.md

```diff
@@ -0,0 +1,139 @@
+        "imageFormat": "2",
+        "pool": "replicapool"
+      },
+      "provisioner": "rook-ceph.rbd.csi.ceph.com",
+      "reclaimPolicy": "Delete",
+      "volumeBindingMode": "Immediate"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/storageclasses?cluster_id=:cluster_id</code>
+
+Retrieve a list of all storage classes in a given [environment](#administration-environments).
+
+| Required                   | &nbsp;                                                      |
+| -------------------------- | ----------------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the storage classes. |
+
+| Attributes                           | &nbsp;                                                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id` <br/>_string_                   | The id of the storage class. This is the name of the resource                                                                               |
+| `isDefault` <br/>_boolean_           | Whether or not the storage class is the default one                                                                                         |
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
+| `metadata` <br/>_object_             | The metadata of the storage class.                                                                                                          |
+| `parameters` <br/>_object_           | The parameters for the storage provisioner. These are storage provisioner specific and you will likely have to read external documentation. |
+| `provisioner` <br/>_string_          | The provsioner for the storage class                                                                                                        |
+| `reclaimPolicy` <br/>_string_        | The default volume reclaim policy for this storage class. You have a choice between `Reclaim` or `Delete`.                                  |
+| `volumeBindingMode` <br/>_string_    | The default volume binding model for this storage class. You have a choice between ``, `` or ``.                                            |
+<!-------------------- GET A storage class -------------------->
+
+#### Get a storage class
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/storageclasses/default-token-xxxmt/default?cluster_id=a_cluster_id"
+```
```

Comment:
Don't you need the storage class name or id?

Reply:
65353eb48b51bd793bce7f6b2b002d3f2b0a0756

Reply:
Good catch, I changed the path. I copied and paste the call from stateful sets lol which takes a name/namespace. 

It makes more sense just with the name

Reply:
7ef0afa025b0f98cf57e59dd798607d7572622a8